scripts/download-codecov-uploader.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
```

Comment:
how long does this usually take? is it worth it to use the cache considering the time difference and the added boilerplate?

Reply:
61 MB binary

Reply:
caching it in azure vs downloading it from some cdn /shrug

Reply:
It took 4s when I tested this in the pipeline. I was fooled by my very slow internet connection.
I am removing the caching

Reply:
![image](https://user-images.githubusercontent.com/683742/140058762-3a561f1d-4657-45b7-bd51-9672ffdc310f.png)

Reply:
![denmark](https://user-images.githubusercontent.com/683742/140058852-c2a7dd61-6415-4065-96fd-5ac68408c2a2.gif)